crates/re_log_types/src/lib.rs

```diff
@@ -52,7 +52,8 @@ pub use self::data_table::{
 };
 pub use self::index::*;
 pub use self::path::*;
-pub use self::time::{Duration, Time};
+// TODO(paris): TimeZone is also used by Timeline etc. in time_point - should I put it under a different parent / a new top level parent?
```

Comment:
that's fine, we can leave it there for the time being (pun not intended)

Reply:
Great, thanks.